messages/scratchOrgCreate.md

```diff
@@ -25,3 +25,23 @@ Expected 'retry' to be an integer number.
 # WaitValidationMaxError
 
 Expected 'wait' greater than or equal to %s but received %s.
+
+# NoScratchOrgInfoError
```

Comment:
SF style guide asks that error message entries carry an `error.` prefix

Reply:
oh, for libraries, too?  That doesn't seem to be the case anywhere else for core v3.

Reply:
Have a look at my core PR :). @jshackell-sfdc does the message prefixes only apply to plugins?

Reply:
@mdonnalley all the errors in core v3 should be renamed?

Reply:
I don't think libraries need to adhere to the plugin style guide. 

Personally I'd prefer to keep all the error names within sfdx-core consistent